src/pages/AccessibilityStatement.svelte

```diff
@@ -0,0 +1,120 @@
+<script lang="ts">
+  import "../style/main.css";
+  // @ts-expect-error no declarations
+  import { initAll } from "govuk-frontend";
+  import { AlphaBanner, DefaultButton } from "govuk-svelte";
+  import { Header } from "lib/common";
+  import { onMount } from "svelte";
+
+  onMount(async () => {
+    // For govuk components. Must happen here.
+    initAll();
+  });
+</script>
+
+<div class="govuk-width-container govuk-prose">
+  <Header />
+  <AlphaBanner />
+
+  <h1 style="margin-top: 30px">
+    Plan Your Active Travel Schemes Accessibility Statement
+  </h1>
+
+  <p>
+    This accessibility statement applies to the Plan Your Active Travel Schemes
+    tool, which is for sketching and viewing active travel schemes at various
+    stages of the bidding and development process.
+  </p>
+  <p>
+    This website is run by Active Travel England. It requires use of and
+    interactions with maps.
+  </p>
+  <ul>
+    <li>
+      We include multiple options for basemap, allowing the user to decide what
+      works best for them
+    </li>
+    <li>
+      We intend to add user-configurable colour choices for map layers such as
+      boundaries, sketched schemes and trip generators
+    </li>
+    <li>
+      <a href="https://mcmw.abilitynet.org.uk/">AbilityNet</a>
+       has advice on making your device easier to use if you have a disability
+    </li>
+  </ul>
+
+  <h2>How accessible this website is</h2>
+
+  <p>We know some parts of the website are not fully accessible:</p>
+  <p>
+    This website is run by Active Travel England. It requires use of and
+    interactions with maps.
+  </p>
+  <ul>
+    <li>You cannot navigate most of the website using just a keyboard</li>
+    <li>
+      You cannot navigate most of the website using speech recognition software
+    </li>
+    <li>
+      You cannot listen to most of the website using a screen reader (including
+      the most recent versions of JAWS, NVDA and VoiceOver)
+    </li>
+    <li>
+      Colour will be used to convey information such as road width and speed
+      limits, until this is configurable it might present an accessibility
+      issue.
+    </li>
+  </ul>
+
+  <h2>Feedback and contact information</h2>
+  <p>
+    If you need information on this app or to provide further feedback on
```

Comment:
Phrasing is a bit odd with the colon. "on issues, please email <a href....>"
Why the "Email - " bit?